src/app/DIAmeterApplication.cpp

```diff
@@ -73,241 +80,871 @@ int DIAmeterApplication::main(const vector<string>& input_files, const string in
   TideMatchSet::initModMap(pepHeader.nprotterm_mods(), PROTEIN_N);
   TideMatchSet::initModMap(pepHeader.cprotterm_mods(), PROTEIN_C);
 
+  // Output setup
```

Comment:
so overall.  Indentation in Crux is 2 space. You will have to change all of the idents to 2 space from what you currently have.